UnityProject/Assets/Scripts/Systems/Explosions/NodeTypes/ExplosionNode.cs

```diff
@@ -113,9 +115,27 @@ public virtual float DoDamage(Matrix matrix, float damageDealt, Vector3Int v3int
 				// do damage
 				player.ApplyDamageAll(null, damageDealt, AttackType.Bomb, DamageType.Brute, default, TraumaticDamageTypes.NONE, 75);
 			}
+
+			ChangeNodeTemp(matrix, damageDealt, v3int);
+
 			return energyExpended;
 		}
 
+		private void ChangeNodeTemp(Matrix matrix, float damageDealt, Vector3Int v3int)
+		{
+			try
+			{
+				if (matrix.ReactionManager != null)
+				{
+					matrix.ReactionManager.ExposeHotspot(v3int, 350 * damageDealt, true);
+				}
+			}
+			catch (Exception e)
+			{
+				Loggy.Log("[ExplosionNode/DoDamage] - Something went wrong while trying to change tile temperature:\n "+ e.ToString());
```

Comment:
if something went wrong then use the ``Error`` message instead of plain log

Reply:
done